mutants/clarity/mutants.out/caught.txt

```diff
@@ -0,0 +1,54 @@
+clarity/src/vm/types/signatures.rs:1923: replace <impl Display for StringUTF8Length>::fmt -> fmt::Result with Ok(Default::default())
```

Comment:
Added it in a previous commit, but removed it from the `.gitignore` because I wanted to share how it is supposed to look and the differences in output between the before fix and after fix. I'll create another commit on top of it having it removed.

The main problem related to these tests is that you would want to have the output saved somewhere in order to check if you want to pass its tests for specific functions. And without having it saved beforehand, would result in hours of waiting to run these.

Reply:
So the point is to diff these known output files with the output of each run? Just make it clear then in the naming/docs that these are reference files, not output